src/layout/components/NavBarHorizontal/index.tsx

```diff
@@ -23,20 +23,20 @@ const NavBarHorizontal: React.FC<NavBarHorizontalProps> = (props) => {
     <>
       <nav className="navbar-horizontal">
         <NavLink exact activeClassName="selected" to="/">
-          <NavTileItem icon={DashboardIcon} text="Dashboard" />
+          <NavTileItem icon={DashboardIcon} />
         </NavLink>
         <NavLink activeClassName="selected" to="/leaderboard">
-          <NavTileItem icon={LBIcon} text="Leaderboard" />
+          <NavTileItem icon={LBIcon} />
         </NavLink>
         <NavLink exact activeClassName="selected" to="/profile">
-          <NavTileItem icon={ProfileIcon} text="Profile" />
+          <NavTileItem icon={ProfileIcon} />
         </NavLink>
         <NavLink activeClassName="selected" to="/discord">
-          <NavTileItem icon={DiscordIcon} text="Discord" />
+          <NavTileItem icon={DiscordIcon} />
         </NavLink>
         {isAdmin && (
           <NavLink activeClassName="selected" to="/admin">
-            <NavTileItem icon={AdminIcon} text="Admin" />
+            <NavTileItem icon={AdminIcon} />
```

Comment:
How do these changes tie in to redux stuff?

Reply:
The `text` prop was unused in the `NavTileItem` component so I removed it. I think it came up when I was resolving lint warnings.